### ISSUE TITLE
Wiki Markup:  Consider a new markup language or major breaking changes to the existing one

### ISSUE THREAD

**Issue by nattthebear** [adelikat/tasvideos#219]
In short, the wiki markup is a mess, and it frustrates editors and developers alike.  Could we do better with something else?

It would most certainly be a different flavour we applied to new pages only, or new editions of an existing page if the editor opts in.

**Comment by vadosnaprimer** [comment]
A huge part should be convertable, having the old wiki mostly replicated (with more or less effort required), and if something breaks after that, wiki editors wouldn't feel so overwhelmed as if **everything** had to be ported manually :D

**Comment by nattthebear** [comment]
> It would most certainly be a different flavour we applied to new pages only, or new editions of an existing page if the editor opts in.

Something we discussed at the time as a possibility but didn't make it into the ticket, is a full automatic conversion of all pages.  Assuming the AST remains the same, since we have `Old Markup => AST`, and we'd be building `New Markup => AST`, if we also build `AST => New Markup`, then we can just, as an update step, convert every single revision of every single page to use the new markup by `Old Markup => AST => New Markup`.

**Comment by adelikat** [comment]
Yeah, that is a good idea.  I really look forward to getting to the point where we can design a new system and convert the old stuff
